cfgov/data_research/scripts/source_to_dump.py

```diff
@@ -0,0 +1,138 @@
+from __future__ import unicode_literals
+
+import datetime
+from dateutil import parser
+import logging
+import sys
+
+import unicodecsv
+
+from data_research.models import MortgageDataConstant, County
+from data_research.mortgage_utilities.s3_utils import (
+    read_in_s3_csv, S3_SOURCE_BUCKET, S3_SOURCE_FILE)
+from data_research.mortgage_utilities.fips_meta import validate_fips
+from data_research.mortgage_utilities.sql_utils import (
+    assemble_insertions,
+    DROP_AND_CREATE_STRING,
+    UNLOCK_STRING)
+
+DUMP_SLUG = '/tmp/mp_countydata'
```

Comment:
Are the resulting files here used later? Would it be better to take an argument for a filename to output to, or generate a filename with `tempfile.mkstemp()` and return it?

Reply:
The next step in the pipeline will load this tmp file and then remove it.

Reply:
So could the pipeline provide the filename to the script? Seems like the script should be agnostic about where it puts the data, and that makes the pipeline fully responsible for the location and cleanup of the file, and we wouldn't have to manage the filename in multiple places in the future if needed.